content/automate/ManageIQ/System/CommonMethods/MiqAe.class/__methods__/weightedupdatestatus.rb

```diff
@@ -0,0 +1,140 @@
+module ManageIQ
```

Comment:
@fdupont-redhat 
Didn't this file get moved when you moved the MiqAe to MiqAe.class. This is not a new file.
Have you rebased after your PR got merged?

Reply:
I have made changes to handle generic error messages.

Reply:
The existing files are named with underscore characters: `weighted_update_status.rb` and `weighted_update_status.yaml`.  This PR is adding additional files without underscores in the names.

@fdupont-redhat Please correct.

Reply:
You're correct. I had renamed the files for consistency with the method name, which has no underscore.